docker/federation/docker-compose.yml

```diff
@@ -18,7 +18,6 @@ services:
       - lemmy_gamma
       - pictrs_gamma
       - iframely
-    restart: "always"
 
```

Comment:
Having all these instances run on startup locally was really annoying.

Reply:
You mean it takes too long to start Lemmy? For that it should help to remove iframely from depends_on, anyway its not needed for Lemmy to start.

Reply:
Nah just that on restarting my docker, it automatically restarts all these, and that was kinda annoying.